setup.cfg

```diff
@@ -44,6 +44,7 @@ install_requires =
   gputil==1.4.0
   psutil==5.9.5
   clu==0.0.7
+  matplotlib==3.7.2
```

Comment:
nit: can you make it matplotlib>=3.7.2 because we really don't care if it's the exact version or not because all this does is plot?

Reply:
fixed